java/j2ee.eclipselink/nbproject/project.properties

```diff
@@ -38,3 +38,6 @@ release.external/jakarta.persistence-2.2.3.jar=modules/ext/eclipselink/jakarta.p
 release.build/jakarta.persistence-2.2.3-doc.zip=modules/ext/docs/jakarta.persistence-2.2.3-doc.zip
 
 extra.module.files=modules/ext/docs/jakarta.persistence-2.2.3-doc.zip
+
+# There are some unexplored dependencies (Java Mail, and JAX-RS API ?)
```

Comment:
```suggestion
# There are some missing dependencies (Java Mail, and JAX-RS API ?)
```

I'm wondering whether it would make sense to disable sigtest here as the API definition is much to broad. ecpliselink has a huge set of `internal` packages, which I would not consider API.